swift-generator/pom.xml

```diff
@@ -93,5 +93,31 @@
       <groupId>io.airlift</groupId>
       <artifactId>log</artifactId>
     </dependency>
+    
+    <dependency>
+      <groupId>com.facebook.swift</groupId>
+      <artifactId>swift-test-utils</artifactId>
+      <scope>test</scope>
+    </dependency>
+    
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>
+      <scope>test</scope>
+    </dependency>
+    
   </dependencies>
+  
+  <build>
+      <plugins>
+          <plugin>
```

Comment:
okay, sorry, I'm an idiot ... I somehow misread this as being the swift maven plugin and then got predictably confused

overall changes lgtm